zio/ndjsonio/reader.go

```diff
@@ -49,6 +49,10 @@ func NewReader(reader io.Reader, zctx *resolver.Context, opts ReaderOpts, filepa
 	if err != nil {
 		return nil, err
 	}
+	_, err = zctx.LookupTypeAlias("port", zng.TypeUint16)
```

Comment:
There don't seem to be any tests that exercise this.  If I comment them out, all tests pass.  But I'm pretty sure the typings stuff needs this and you are totally correct that the typings schema policy should define what zenum and port are rather than being hardcoded here.  I will add the comment you suggest.  I can't find an issue where to hang this so I will create a new one but it will probably be redundant with the larger task of imrpoving schema maps.  #1427